app/build.gradle.kts

```diff
@@ -168,6 +168,8 @@ dependencies {
     implementation(libs.androidx.work)
     implementation(libs.androidx.recyclerview)
 
+    implementation(libs.kotlinx.collections)
```

Comment:
This line shouldn't be needed because the library is already added in the `libs.bundles.kotlin`.